docs/installation/docker-compose.md

```diff
@@ -138,57 +132,140 @@ docker-compose up -d
 
-To log in:
+By default, the credentials to login are
 
 * username: **admin**
 * password: **password**
 
-!!! Note Docker logs and Docker Compose logs
-    `docker logs -tf isle-dc_drupal_1` shows Docker logs for the "drupal" container, which will continue to have new log entries as long as the drupal box is still starting up. When it gets to `confd using 'env' backend`, you're done. `docker-compose logs` is like a firehose, showing the log messages from all containers.
+If one of your containers fails to launch, you'll need to check its logs using `docker-compose logs -tf SERVICENAME`.  For example, to see Drupal's logs, use `docker-compose logs -tf drupal`.  Usually, when you see `confd using 'env' backend`, the service has booted up successfully.  If you want to see all logs, just run `docker-compose logs`. 
 
-## Spinning down your Islandora site
+### Shutting down your Islandora site
 
 To shut down the containers without destroying your site, use `docker-compose down`. To also destroy your "state" (i.e. your content, your database, your files), use `docker-compose down -v`.
 
-## Editing Code in Docker
+### Bringing your Islandora site back up
 
-If you used `make local` then the drupal root folder is in a new directory in the isle-dc folder named `codebase`. This is live and editable in whatever development environment you would like.  If you just did `make demo`, you will need to spin down your containers with `-v` to destroy your state before starting a new one with `make local`.
+Use `docker-compose up -d` to bring a site back up. You should _not_ run `make local` or `make demo` again unless you used `-v` when shutting down the site.
 
-Editing code for the back-end processes (alpaca, milliner, etc) is more complicated. Please ask on the #isle Slack channel and help us improve this documentation!
+## Useful Make Commands
 
-### Testing a Pull Request
+There's a lot of other useful utilities available to you from within the `isle-dc` folder.
 
-Islandora modules in the `codebase/web/contrib/modules` folder are already set up with `git` and the `origin` remote is the canonical Islandora repository. You can follow the command-line instructions for testing pull requests available on Github. When finished, don't forget to `git checkout main` (or the default branch if not named main) so you can pull new code.
+### Rebuild docker-compose.yml
 
-## Updating ISLE
+`make -B docker-compose.yml` will rebuild your `docker-compose.yml` file based on configuration in your `.env` file.  `make demo` and `make local` will automatically do this for you, but if you change configuration at a later point in time, you'll need to run this command to see your changes reflected.
+
+### Config Export
+
+`make config-export` will export your site's configuration to your site's config sync directory.
+
+### Config Import
+
+`make config-import` will export import site's configuration from your site's config sync directory.
```

Comment:
`will export import site's` should probably say `will import site's`

Should this also point out where the config/sync directory is?  namely, it's in `config/sync` in your drupal's www folder.